browser/src/map/handler/Map.Keyboard.js

```diff
@@ -591,33 +698,33 @@ L.Map.Keyboard = L.Handler.extend({
 		}
 
 		switch (e.keyCode) {
-		case 51: // 3
+		case this.keyCodes.NUM3[DEFAULT]: // 3
 			if (this._map.getDocType() === 'spreadsheet') {
 				app.socket.sendMessage('uno .uno:SetOptimalColumnWidthDirect');
 				app.socket.sendMessage('commandvalues command=.uno:ViewRowColumnHeaders');
 				return true;
 			}
 			return false;
-		case 53: // 5
+		case this.keyCodes.NUM5[DEFAULT]: // 5
 			if (this._map.getDocType() === 'spreadsheet') {
 				app.socket.sendMessage('uno .uno:Strikeout');
 				return true;
 			}
 			return false;
-		case 67: // 'C'
-		case 88: // 'X'
-		case 99: // 'c'
-		case 120: // 'x'
-		case 91: // Left Cmd (Safari)
-		case 93: // Right Cmd (Safari)
+		case this.keyCodes.C[DEFAULT]: // 'C'
+		case this.keyCodes.X[DEFAULT]: // 'X'
+		case this.keyCodes.C[MAC]: // 'c' Since keydown+c as different value in mac so i had to update the mapping in keyCodes 
+		case this.keyCodes.X[MAC]: // 'x' same reason as above
+		case this.keyCodes.LEFTWINDOWKEY: // Left Cmd (Safari)
+		case this.keyCodes.RIGHTWINDOWKEY: // Right Cmd (Safari)
```

Comment:
it was ` 93: // Right Cmd (Safari)`
but `RIGHTWINDOWKEY` is  `92`

Reply:
Oh I missed it. Will do it.

Reply:
done